EliteDangerous/JournalEvents/JournalScan_Display.cs

```diff
@@ -436,7 +436,14 @@ public string MassEMText()
                 information.Append(" (Discovered)".T(EDCTx.JournalScanInfo_DIS));
                 if (showvalues)
                 {
-                    information.Append(' ').Append(ev.EstimatedValueFirstMappedEfficiently.ToString("N0")).Append(" cr");
+                    information.Append(' ').Append((ev.EstimatedValueFirstMappedEfficiently > 0 ? ev.EstimatedValueFirstMappedEfficiently : ev.EstimatedValueBase).ToString("N0")).Append(" cr");
+                }
+            }
+            else if (js.WasDiscovered == false && js.IsStar)
+            {
+                if (showvalues)
+                {
+                    information.Append(' ').Append(ev.EstimatedValueFirstDiscovered.ToString("N0")).Append(" cr");
```

Comment:
So, i know this is a bit pedantic, but for scans before 3.2, this would show zero, as FirstDiscovered would not be set (see EstimatedValues). I suggest a test in which variable you pick, like 439.

Reply:
Fixed, thanks for reminding me of ancient data.